Apps/phtaniumrest/readme.html

```diff
@@ -3,26 +3,87 @@
 
   Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
 -->
-<p><b>Run Query Guidelines</b>
+<h2>Asset Configuration</h2>
+<li><b>Consider question results complete at (% out of 100)</b></li>
+<ul>
+  <li>This asset parameter refers to the amount of question results you wish to be returned by actions <b>run query</b> and <b>list processes</b> before considering the action to be completed.
+      The time spent returning your results is dependent on how much data you have on your Tanium instance and you may want your action to end with a certain percentage threshold instead of waiting for Tanium to return
+      100% of the results. This parameter is also configurable in your Tanium instance's preferences.</li>
+</ul>
+<h2>Run Query</h2>
```

Comment:
Recommend we rename this to give more context.  
"How to use Run Query Action" and apply a similar update to the other action headings